sdks/java/extensions/sql/zetasql/src/test/java/org/apache/beam/sdk/extensions/sql/zetasql/ZetaSQLDialectSpecTest.java

```diff
@@ -3661,7 +3661,6 @@ public void testExtractTimestampThrowsOnMicrosecondNotSupported() {
 
   /** Only sample scenarios are covered here. Excessive testing is done via Compliance tests. */
   @Test
-  @Ignore("ZetaSQL does not support EnumType to IdentifierLiteral")
```

Comment:
How could this test pass without running `BeamZetaSqlCalcRel`? My understanding is that the changes in this PR is only used in `BeamZetaSqlCalcRel`. Is this not correct?

Reply:
BigQuery predicate push-down also makes use of the unparsing.

Reply:
I think it could pass because ZetaSQL somehow starts to support EnumType.

Reply:
Limited support for EnumType was added but the tests weren't turned on.